datalad/interface/annotate_paths.py

```diff
@@ -177,7 +177,9 @@ def get_modified_subpaths(aps, refds, revision, recursion_limit=None,
 
     # life is simple: we diff the base dataset
     modified = []
-    for r in refds.diff(
+    from datalad.interface.diff import Diff
```

Comment:
I would appreciate having a comment here like 

```
# Diff.__call__ is used instead of `refds.diff` since `interface.diff` is being deprecated
```

Reply:
Good point.  Added.